app/_development-tools/controllers.js

```diff
@@ -0,0 +1,27 @@
+const { uniq } = require('lodash')
```

Comment:
`_development-tools` does not thrill me, however it does make it obvious that its a "non-functional requirement" module.

Only alternative to this I can think of is ripping this out as a separate `npm module` and including it that way - but I think it's too early for us to know what that would look like just yet.

Reply:
Yep, it didn't thrill me either...I also toyed with `./dev`, `./development-only`, `./_tools`....the prefix sort of helped set it out above the rest of the "real" apps. But maybe that's not necessary as we don't do that with the component app which is only available in dev or on certain environments.

What about simple `./tools`?